charts/collabora-online/0.0.1/questions.yaml

```
@@ -0,0 +1,314 @@
+      default: Recreate
+      enum:
+        - value: "RollingUpdate"
+          description: "Create new pods and then kill old ones"
+        - value: "Recreate"
+          description: "Kill existing pods before creating new ones"        
+
+  # Service Configuration
+  - variable: service
+    group: "Networking"
+    label: "Configure Network"
+    schema:
+      type: dict
+      attrs:
+        - variable: type
+          label: "Service type"
+          schema:
+            type: string
+            default: "ClusterIP"
+            enum:
+              - value: "NodePort"
+                description: "NodePort"
+              - value: "ClusterIP"
+                description: "ClusterIP"
+            show_subquestions_if: "NodePort"
+            subquestions:
+              - variable: port
+                label: "Port configuration"
+                schema:
+                  type: dict
+                  attrs:
+                    - variable: port
+                      label: "container port"
+                      schema:
+                        type: int
+                        default: 9980
+                        editable: false
+                    - variable: nodePort
+                      label: "Node Port to expose for UI"
+                      schema:
+                        type: int
+                        min: 9000
+                        max: 65535
+                        default: 30980
+                        required: true
+                              
+  # environmentVariables Configuraiton
+  - variable: env
+    group: "Environment Variables"
+    label: "Environment Variables Configuration"
+    schema:
+      type: dict
+      required: true
+      attrs:
+        - variable: domain
+          label: "Domains will be using collabora"
+          description: 'Use backslash "\" before dots ".". Use pipe "|" to separate multiple domains'
+          schema:
+            type: string
+            default: 'nextcloud\.domain\.tld|othernextcloud\.domain\.tld'
+            required: true
+        - variable: username 
+          label: "Username for WebUI"
+          schema:
+            type: string
+            default: "admin"
+            required: true
+        - variable: password
+          label: "Password for WebUI"
+          schema:
+            type: string
+            private: true
+            default: "changeme"
+            required: true
+        - variable: dictionaries
+          label: "Dictionaries to use, leave empty to use all"
+          schema:
+            type: string
+            default: "de_DE en_GB en_US es_ES fr_FR it nl pt_BR pt_PT ru"
+        - variable: extra_params
+          label: "Extra Parameters to add"
+          description: 'e.g. "–o:welcome.enable=false", See more on /etc/loolwsd/loowsd.xml. Separate params with space'
+          schema:
+            type: string
+            default: "-o:welcome.enable=false -o:user_interface.mode=notebookbar"        
```

You might also want to set:
`--o:ssl.termination=true --o:ssl.enable=false` 
because internally we don't use SSL (at least for now).

```suggestion
            default: "-o:welcome.enable=false -o:user_interface.mode=notebookbar -o:ssl.termination=true -o:ssl.enable=false"        
```

---

> @stavros-k Awesome, then you can cut the storage and persistance options and finish up the last feedback :)

Done 

> Also: Make sure to test out the ingress and squash all commits

Is it as simple as Enabling ingress and putting the domain name in the Host?
And wait a couple of minutes? Have no experience with traefik. Only with nginx

About squashing, i'm reading on how to do it first :D Should find out later in the day